scale/project/project.go

```diff
@@ -0,0 +1,103 @@
+package project
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+type Project struct {
+	Path string
+	Name string
+
+	client *client.Client
+}
+
+type UpOpts struct {
+	Scale map[string]int
+	Wait  time.Duration
+}
+
+func New(fullpath string, client *client.Client) Project {
```

Comment:
Maybe just declare it as `New(path, name string, client *client.Client) Project` instead of having his split/join logic?

Reply:
this way it is protected from renaming

Reply:
actually will do it this way